src/app/campaign/store/effects/campaign.effects.ts

```diff
@@ -64,6 +65,10 @@ export class CampaignEffects {
               updatedFlightDocs: { [id: number]: HalDoc };
               deletedFlightDocs: { [id: number]: HalDoc };
             }) => {
+              // save off new campaign id in case campaign create succeeded but flight creation fails
```

Comment:
not my most genius idea for error handling, but solving the navigation problem and where to show the error is all we needed to do, so it works 🤷